packages/contentstack-migration/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentstack/cli-migration",
-  "version": "1.1.1",
+  "version": "1.2.0",
   "author": "@contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
```

Comment:
@aman19K cli-command and cli-utilities is at it's latest verion 1.2.0, why you are still using 1.1.0?

Reply:
@netrajpatel version updated.